app/helpers/PermissionHints.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace App\Helpers;
+
+use Nette\Reflection\ClassType;
+use Nette\Reflection\Method;
+use Nette\StaticClassException;
+use Nette\Utils\Strings;
+
+class PermissionHints {
+  public function __construct() {
+    throw new StaticClassException();
+  }
+
+  public static function generate($aclModule, $subject) {
```

Comment:
This method is in dire need of doc comment. I must have read it twice and slowly to understand it completely.